src/Console/Command/ReadmeCommand.php

```diff
@@ -46,14 +46,23 @@ protected function execute(InputInterface $input, OutputInterface $output)
 PHP Coding Standards Fixer
 ==========================
 
-The PHP Coding Standards Fixer tool fixes *most* issues in your code when you
-want to follow the PHP coding standards as defined in the PSR-1 and PSR-2
-documents and many more.
+The PHP Coding Standards Fixer (PHP CS Fixer) tool fixes your code to follow standards;
+whether you want to follow PHP coding standards as defined in the PSR-1, PSR-2, etc.,
+or other community driven ones like the Symfony one.
+You can **also** define your (teams) style through configuration.
+
+It can modernize your code (like converting to PHP 5.4 `short array syntax`) and (micro) optimize it.
```

Comment:
for a lot of ppl array syntax is style only, I would give other example. maybe `**` operator ?